codelists/+partyRole.csv

```diff
@@ -5,3 +5,11 @@ processContactPoint,Process contact point,A contact point dedicated to this cont
 reviewContactPoint,Review contact point,The service from which information about the review procedure can be obtained.
 selectedParticipant,Selected participant,A party that has already been selected to participate in the design contest.
 informationService,Information service,An organization that can provide information regarding specific aspects of the contracting process.
+procurementServiceProvider,Procurement service provider,An organization that provides procurement services to contracting authorities.
+eSender,TED eSender,An organization that provides data transfer services to contracting authorities.
+leadBuyer,Lead buyer,The lead buyer in a group of buyers.
+awardingCentralPurchasingBody,Awarding central purchasing body,A central purchasing body awarding public contracts or concluding framework agreements for works supplies or services intended for other buyers.
+acquiringCentralPurchasingBody,Acquiring central purchasing body,A central purchasing body acquiring supplies and/or services intended for other buyers.
```

Comment:
Should we align the language here with OCDS and refer to "goods, services or works"? The definitions you've used here do align with what's in the EU descriptions quoted in https://github.com/open-contracting/standard/issues/1180#issue-792561618 so happy to leave them as they are if that'd be better as the codes in this extension are explicitly for EU fields.

Reply:
Good spot. I think there's a couple of other problems with descriptions as drafted:

* There's a missing comma in "works supplies or services" in the description of 'awardingCentralPurchasingBody'
* There's a difference in the procurement categories listed in the description of 'awardingCentralPurchasingBody' (works supplies or services) and 'acquiringCentralPurchasingBody' (supplies and/or services). I don't think the use of these codes is intended to communicate anything about the object of the contracting process.

Therefore, I've replaced both with 'goods, services or works'.